chain-impl-mockchain/src/testing/e2e/fees.rs

```diff
@@ -239,3 +244,93 @@ fn verify_total_funds_after_transaction_with_fee(fee: u64) {
         .pots()
         .has_fee_equals_to(&Value(fee));
 }
+
+#[quickcheck]
+pub fn vote_cast_fees(constant_fee: u64, coefficient: u64, certificate: u64) {
+    let expected_fees = constant_fee + coefficient + certificate;
+    let favorable = Choice::new(1);
+    let voting_token = TokenName::try_from(vec![0u8; TOKEN_NAME_MAX_SIZE]).unwrap();
```

Comment:
i think we got `TestGen::token_name()` static method for this purpose

Reply:
Not working for this purpose as TestGen::token_name() returns an incompatible type of token.
As discussed I created a ticket to track this desiderata: https://input-output.atlassian.net/browse/NPG-1217